_sitegen/controllers/COSummary.js

```diff
@@ -103,6 +103,23 @@ function showConfirmation(order) {
                 dataObj = orderConfirmationData.getData(order, exchangeID);
                 serviceCallResult = klaviyoUtils.trackEvent(exchangeID, dataObj, klaviyoUtils.EVENT_NAMES.orderConfirmation, order.customerEmail);
             }
+
+            if ('KLEmailSubscribe' in session.custom || 'KLSmsSubscribe' in session.custom) {
+                var email = session.custom.KLEmailSubscribe ? order.customerEmail : false;
```

Comment:
We still want to pass along the email field even if `KLEmailSubscribe` is false and same goes for phone number